### PR TITLE
Add data operation ability on AbstractTableCommand

### PR DESCRIPTION
**Pull request by JySongWithZhangCe**
Related with https://github.com/Alluxio/alluxio/issues/12690

### CODE REVIEW

**Comment by JySongWithZhangCe** [comment]
@gpang Could you give some suggestions? Thx:)

File: table/shell/src/main/java/alluxio/cli/table/command/AbstractTableCommand.java [hunk 1]
```diff
@@ -36,8 +39,11 @@
    *
    * @param conf the alluxio configuration
    * @param client the client interface which can be used to make RPCs against the table master
+   * @param fsContext the filesystem of Alluxio
    */
-  public AbstractTableCommand(AlluxioConfiguration conf, TableMasterClient client) {
+  public AbstractTableCommand(AlluxioConfiguration conf, TableMasterClient client,
+      @Nullable FileSystemContext fsContext) {
```
Comment (gpang) [inline table/shell/src/main/java/alluxio/cli/table/command/AbstractTableCommand.java:45]: Why is nullable supported here? Shouldn't all commands be able to take a non-null context?

**Review by gpang — COMMENTED** [review]
@JySongWithZhangCe I left a minor question.

File: table/shell/src/main/java/alluxio/cli/table/command/AbstractTableCommand.java (hunk 1 above)
Comment (JySongWithZhangCe) [reply table/shell/src/main/java/alluxio/cli/table/command/AbstractTableCommand.java]: I agree

**Comment by JySongWithZhangCe** [comment]
@alluxio-bot , test this please

**Comment by JySongWithZhangCe** [comment]
> @JySongWithZhangCe I left a minor question.

I've fixed code on your suggestion

**Comment by JySongWithZhangCe** [comment]
@gpang

**Review by gpang — APPROVED** [review]
LGTM

**Comment by JySongWithZhangCe** [comment]
> LGTM

Thx:)

**Comment by JySongWithZhangCe** [comment]
@apc999 Could you please merge this PR? Thx:)

**Comment by bradyoo** [comment]
alluxio-bot, merge this please.

**Comment by LuQQiu** [comment]
@bradyoo Does this PR need to be cherry-picked to branch-2.5?

**Comment by bradyoo** [comment]
@LuQQiu shouldn't be necessary.

**Pull request merged**